package.json

```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "@headlessui/react": "^1.4.3",
-    "gatsby": "^4.5.4",
+    "gatsby": "^4.17.1",
```

Comment:
The version of gatsby we are using should be working fine was there any reason to update